pkg/backends/backend.go

```diff
@@ -80,9 +80,9 @@ type Registrar func(map[string]http.Handler)
 
 // New returns a new Backend
 func New(name string, o *bo.Options, registrar Registrar,
-	router http.Handler, cache cache.Cache) (Backend, error) {
+	router http.Handler, c cache.Cache) (Backend, error) {
```

Comment:
there's a rule that warns against shadowing package imports. I'm not entirely sold on its importance. When it becomes a problem, its _usually_ obvious (usually based on code editor autocomplete, in my experience).

It would reduce this PR by a little, and lessen up a possibly needless restriction.

Thoughts?

Reply:
yeah this seems like a too-late opinion to apply, going to revert